src/utils/notifications.ts

```diff
@@ -69,7 +69,7 @@ export const raiseNativeNotification = (
 
   if (notifications.length === 1) {
     const notification = notifications[0];
-    title = `Gitify - ${notification.repository.full_name}`;
```

Comment:
I see how this would be better looking on Windows, but on macOS this is important to help identify the app. Could you please make this platform specific?

Reply:
```js
title = `${process.platform !== 'win32' ? 'Gitify - ' : ''}${notification.repository.full_name}`;
```
would that be ok?

Reply:
yes - let's go with that for now!

Reply:
done in [8869d69](https://github.com/manosim/gitify/pull/539/commits/8869d69aee8b20dab0666a6aef2daa6a6bbd5c82)